components/wallet/WalletContextPanel.tsx

```
@@ -0,0 +1,100 @@
+  environment,
+  mintId,
+  fanoutMints,
+  selectedFanoutMint,
+  onSelectMint,
+  className
+}) => {
+  return (
+    <Card elev={1} surface="subtle" className={cn('space-y-8', className)} id="token-selection">
+      <CardHeader heading="Token & Addresses" subtitle="Select an asset and view key on-chain addresses for the treasury." />
+      <section aria-labelledby="token-select-label" className="space-y-3">
+        <label id="token-select-label" className="text-[13px] font-medium text-gray-300 block">Token Selection</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)